### DISCUSSION TITLE
Custom Button on Site View not working

### DISCUSSION THREAD

**Discussion by peterdanielmyers** [joomengine/Joomla-Component-Builder#946] [Extention Issues] [answered]
I have a Site View with a Custom Button. When I click the button, nothing happens. I've tried inserting all sorts of code into the controller method, and it seems that the method is not being called.

Is there anything I might be missing that could be causing this? Can anyone point me in the direction of where I should be looking to try and debug?

Sorry for such an open question, I'm flummoxed.
——————————
EDIT: So, inspecting the Javascript errors, I get this:

"Uncaught TypeError: Cannot read properties of null (reading 'task')"

**Comment by Llewellynvdm** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/5607939/214451979-f5c77c4f-2628-4f5b-8d1e-2d1d9cb2d831.png)

All bug related issues please use https://git.vdm.dev/joomla/Component-Builder/issues as I will not respond to bugs, or possible bugs over here, thanks!
